### PR TITLE
OEA-452

### PR DESCRIPTION
**Pull request by apigram**
Fixed issue where Trial PI could not be changed due to a reference to a nonexistent id POST value in the access control check (actually trial_id).
Users that do not have trial management permissions are no longer able to change the PI of a trial.
Made PI radio buttons read-only if the user does not have manage permissions on the trial.

### CODE REVIEW

File: views/trial/permissions.php [hunk 1]
```diff
@@ -10,301 +10,302 @@
...
-      </div>
-
-        <?php if ($canManageTrial): ?>
-          <h2>Share with another user:</h2>
-          <div class="row field-row">
-            <div class="large-6 column">
-              <div class="row field-row">
-                <div class="large-3 column">
-                    <?php echo CHtml::activeLabel($newPermission, 'user'); ?>
+    <div class="large-9 column">
```
Comment (marshl) [inline views/trial/permissions.php]: The HTML formatting seems to be 4 space. It should be 2 space

File: views/userTrialPermission/_view.php [hunk 2]
```diff
@@ -7,7 +7,12 @@
 
 <tr data-permission-id="<?php echo $data->id; ?>">
     <td>
-        <?php echo CHtml::radioButton('pi_user_id', $data->trial->pi_user_id === $data->user_id, array('class' => 'pi_selector')); ?>
+        <?php
+            if ($canManageTrial) {
```
Comment (marshl) [inline views/userTrialPermission/_view.php]: Just do this check in the htmlOptions. There is no point duplicating code too much.

Comment (apigram) [reply views/userTrialPermission/_view.php]: Fair point. I forgot we can do that.

Comment (apigram) [reply views/userTrialPermission/_view.php]: Actually, this doesn't work. Either the property is specified or it isn't. A True/false check changes nothing

Comment (marshl) [reply views/userTrialPermission/_view.php]: Fair enough. It's a shame you can't do it on the fly

File: views/trial/permissions.php (hunk 1 above)
Comment (apigram) [reply views/trial/permissions.php]: That might be a change as of a recent PHPStorm update. Pretty sure I'm still using the styling that the UK gave us.

Comment (marshl) [reply views/trial/permissions.php]: The most recent UK standards seem to use 2,/2/4 for html spacing. Try a re-import to see if it fixes it (or just set them manually)

Comment (apigram) [reply views/trial/permissions.php]: Thats definitely how it originally was, but for some reason re-importing the settings changes absolutely nothing in PHPStorm.

**Comment by apigram** [comment]
@marshl @Zhaolian-Zhou Bump

**Pull request merged**